### PR TITLE
feat: `--create-ticket` flag to send error reports to sentry

### PR DESCRIPTION
**Pull request by sweatybridge**
## What kind of change does this PR introduce?

feature

## What is the new behavior?

Reports error to sentry when any commands are with `--create-ticket` flag. Helps to collect diagnostics information.
- [x] server side rate limiting
- [x] add stacktrace to all errors

Going forward, any errors from external packages should be wrapped in `errors.New(err)` using [go-errors](https://github.com/go-errors/errors).

## Additional context

Eg: https://supabase.sentry.io/issues/4734608486/events/e250b9cc83c245d1aa520f1e3ece5017

### CODE REVIEW

**Comment by soedirgo** [comment]
Telemetry is a [very](https://news.ycombinator.com/item?id=34707583) [sensitive](https://news.ycombinator.com/item?id=35974653) [topic](https://news.ycombinator.com/item?id=27727150), even for anonymous info. IMO we should make this opt-in by prompting on first run instead of opt-out.

File: .goreleaser.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ builds:
       - linux_amd64
       - linux_arm64
       - windows_amd64
+      - windows_arm64
```
Comment (soedirgo) [inline .goreleaser.yml:17]: I'd rather not support another platform when there's no one in the team with a Windows ARM machine. I don't see anyone requesting for it either.

**Review by soedirgo — APPROVED** [review]
Just a note re: Windows ARM, otherwise lgtm 👍

File: .goreleaser.yml (hunk 1 above)
Comment (sweatybridge) [reply .goreleaser.yml:17]: This is the ticket which requests for windows arm support https://github.com/supabase/cli/issues/1315. Will move to a separate PR though.

**Pull request merged**